examples/shared/src/main/scala-2/examples/SmartTypes.scala

```diff
@@ -64,5 +64,6 @@ object SmartTypes extends App {
   }
   type Email = Email.Type
   val email: Email = Email("test@test.com")
+
```

Comment:
I don't mind adding this new line, but I just want to point it out :wink:

Reply:
lol. I didn't even notice. That's my OCD 😛, I find myself adding spacing before after the start/end braces.

Reply:
Weird about the dots. I blame IntelliJ! Will fix ;)

Reply:
<img width="181" alt="CleanShot 2022-07-27 at 21 18 24@2x" src="https://user-images.githubusercontent.com/7587245/181399646-3304570a-bc4e-4e86-9d99-b459bed23423.png">
<img width="213" alt="CleanShot 2022-07-27 at 21 18 30@2x" src="https://user-images.githubusercontent.com/7587245/181399651-fee9be64-590d-4f23-9cd0-63ac7a9f841a.png">

ha, yeah. they look the same because of how IntelliJ renders this (though the correct one is yellow).